tags/Cliengo/Cliengo Chatbot/Cliengo.tpl

```diff
@@ -0,0 +1,191 @@
+      "key": {
+        "publicId": "logging",
+        "versionId": "1"
+      },
+      "param": [
+        {
+          "key": "environments",
+          "value": {
+            "type": 1,
+            "string": "debug"
+          }
+        }
+      ]
+    },
+    "isRequired": true
+  },
+  {
+    "instance": {
+      "key": {
+        "publicId": "access_globals",
+        "versionId": "1"
+      },
+      "param": [
+        {
+          "key": "keys",
+          "value": {
+            "type": 2,
+            "listItem": [
+              {
+                "type": 3,
+                "mapKey": [
+                  {
+                    "type": 1,
+                    "string": "key"
+                  },
+                  {
+                    "type": 1,
+                    "string": "read"
+                  },
+                  {
+                    "type": 1,
+                    "string": "write"
+                  },
+                  {
+                    "type": 1,
+                    "string": "execute"
+                  }
+                ],
+                "mapValue": [
+                  {
+                    "type": 1,
+                    "string": "ldk"
+                  },
+                  {
+                    "type": 8,
+                    "boolean": true
+                  },
+                  {
+                    "type": 8,
+                    "boolean": true
+                  },
+                  {
+                    "type": 8,
+                    "boolean": true
+                  }
+                ]
+              }
+            ]
+          }
+        }
+      ]
+    },
+    "clientAnnotations": {
+      "isEditedByUser": true
+    },
+    "isRequired": true
+  },
+  {
+    "instance": {
+      "key": {
+        "publicId": "inject_script",
+        "versionId": "1"
+      },
+      "param": [
+        {
+          "key": "urls",
+          "value": {
+            "type": 2,
+            "listItem": [
+              {
+                "type": 1,
+                "string": "https://s.cliengo.com/weboptimizer/*"
+              }
+            ]
+          }
+        }
+      ]
+    },
+    "clientAnnotations": {
+      "isEditedByUser": true
+    },
+    "isRequired": true
+  }
+]
+
+
+___SANDBOXED_JS_FOR_WEB_TEMPLATE___
+
+// Introduzca aquí su código de plantilla.
+var log = require('logToConsole');
+const copyFromWindow = require('copyFromWindow');
+const setInWindow = require('setInWindow');
+const injectScript = require('injectScript');
+const companyId = data.companyId;
+const websiteId = data.websiteId;
+const url = 'https://s.cliengo.com/weboptimizer/'+companyId+'/'+websiteId+'.js';
+
+const setClgo = setInWindow('ldk', function() {
+  const document = copyFromWindow('document');
+  var ldk= document.createElement('script');
+  ldk.type='text/javascript'; 
+  ldk.async=true;
+  ldk.src=url;
+  var s=document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ldk, s);
+  });
```

Comment:
This isn't called, and would try to do what injectScript is already doing. Sandboxed script doesn't allow direct document manipulation. It looks like you can omit this and the require for copyFromWindow and setInWindow.

Reply:
@laziob could you make the change that Eric suggested and update the PR? Thanks.